packages/coil-privacypass-sjcl/src/issuance.ts

```diff
@@ -15,25 +21,31 @@ export const CACHED_COMMITMENTS_STRING = 'cached-commitments'
  * optional commitment version
  * @return {Object} Formatted object for inputs
  */
-export function parseIssueResp(issueResp: any) {
-  let signatures
-  let batchProof
-  let version
+export function parseIssueResp(
+  issueResp: RawIssueResponse | string[]
+): IssueResponse {
+  let signatures: string[]
+  let batchProof: string
+  let version: string
+  let prng: string
   // If this is not an array then the object is probably JSON.
-  if (!issueResp[0]) {
+  if (isRawIssueResponse(issueResp)) {
     signatures = issueResp.sigs
     batchProof = issueResp.proof
     version = issueResp.version
+    prng = issueResp.prng
   } else {
     batchProof = issueResp[issueResp.length - 1]
     signatures = issueResp.slice(0, issueResp.length - 1)
+    // TODO
+    version = (undefined as unknown) as string
```

Comment:
What's going on with these types?

Reply:
A "TODO" WIP
Question is if it break things?
Please feel free to push a fix!

Reply:
Will fix later, thanks!